dbt/include/athena/macros/materializations/snapshots/snapshot.sql

```diff
@@ -248,21 +248,17 @@
   {%- set unique_key = config.get('unique_key') %}
   {%- set file_format = config.get('file_format', 'parquet') -%}
 
+  {{ log('Checking if target table exists') }}
```

Comment:
I did not know this works, I always thought I had to do :
```jinja
{% do log('...') %}
```

Reply:
I followed the docs and this seems to be the recommended way https://docs.getdbt.com/reference/dbt-jinja-functions/log